### DISCUSSION TITLE
MIP-7 / visual differentiation of masterEditionNFTs

### DISCUSSION THREAD

**Discussion by BloodyBee** [phantom/docs#171] [Help & Support]
### Summary

Hi there, BloodyB here!

I'm here to touch base about MIP-7 (which went live on devnet today) and to stress the importance of a strong(er) visual differentiation between a masterEditionNFT & its prints because it would go a long way in preventing artists from burning/transferring their masterEditions accidentally, as a result of this change.

meaning:
with MIP-7 going live on mainnet soon,
loads more artists will suddenly hold masterEditionNFTs for the first time and have no clue what they are. let's make it easier/possible to recognize what's what (masterEd vs print vs collectionNFT), in-wallet, possibly even within the NFT overview already, somehow. :)

Yes, please help us out here in educating artists / holders of masterEditionNFTs at least a tiny bit about what these are, by way of a small indicator + explainer, on-hover, for example. idk, but I'm happy to talk about it!

This could become a real support nightmare for marketplaces otherwise. So, you guys will get to play a vital role in making this be the ecosystem W that it can be. If done right, and in concert. Are you down?

If I can help in any way, shoot!

Cheers from Berlin

### Steps to Reproduce

Try to tell the difference between a masterEdition and the NFTs that were printed from that masterEdition in your wallet overview. You can’t. Can you? That’s part of the problem.

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hello @BloodyBee,

Thank you for writing in!

This is a good feature request for our request board. Please post it [here](https://phantom.canny.io) and ask those interested to upvote the request.
